lib/diagnostic.rb

```diff
@@ -13,7 +13,16 @@
 # and location should only be writable.
 
 ##
-# your code here
+ class Person(name, age, location)
+   def initialize
+     @name = name
+     @age = age
+     @location = location
+   end
+ end
```

Comment:
Correct but your arguments need to go on the initialize method; not on the class. Also you need to have the `attr_accessor/reader/writer`